sofa-ark-parent/core-impl/container/src/main/java/com/alipay/sofa/ark/container/service/biz/BizFactoryServiceImpl.java

```diff
@@ -96,11 +96,10 @@ public Biz createBiz(BizArchive bizArchive) throws IOException {
             .setClassPath(bizArchive.getUrls());
 
         BizClassLoader bizClassLoader = new BizClassLoader(bizModel.getIdentity(),
-            getBizUcp(bizModel.getClassPath()), bizArchive instanceof ExplodedBizArchive);
+            getBizUcp(bizModel.getClassPath()), bizArchive instanceof ExplodedBizArchive || bizArchive instanceof DirectoryBizArchive);
```

Comment:
"import com.alipay.sofa.ark.loader.DirectoryBizArchive" is missing

Reply:
Imported.. Sorry